tutorials/understanding-ibc-denoms/index.md

```diff
@@ -1,23 +1,23 @@
 ---
 parent:
-title: Understand IBC Denoms with Gaia
+title: Understanding IBC denoms with Gaia
```

Comment:
I note that the in page title doesn't include "with Gaia". I didn't want to presume which alternative was preferred, so I've left that element of the title unchanged.

Reply:
Right, forgot about that one. I think we should update and leave off 'with Gaia' as the steps followed are easily extensible to other chains/binaries!